ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -195,7 +227,7 @@ export default function PrivacySettings() {
         }),
       );
     } else {
-      await enableProfileSyncing();
+      await profileSyncingProps.enableProfileSyncing();
```

Comment:
@matteoscurati I don't know if I like this enable here. This is async and we don't show anything to users that this is async.

Instead I kind of want us to just use `profileSyncingProps.setIsProfileSyncingEnabled(true);` which just updates the profile sync state.

When onboarding completes... I think we will be logged in if a user enabled profile syncing?